radiacode-examples/narodmon.py

```diff
@@ -47,20 +48,64 @@ async def send_data(d):
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--bluetooth-mac', type=str, required=True, help='MAC address of radiascan device')
-    parser.add_argument('--connection', choices=['usb', 'bluetooth'], default='bluetooth', help='device connection type')
-    parser.add_argument('--interval', type=int, required=False, default=600, help='send interval, seconds')
+    parser.add_argument(
+        '--bluetooth-mac',
+        type=str,
+        required=False,
+        help='Radiacode Bluetooth MAC address (e.g. 00:11:22:33:44:55). MacOS does not support BT MACs, use Serial Number or UUID instead.',
+    )
+    parser.add_argument(
+        '--bluetooth-serial',
+        type=str,
+        required=False,
+        help='Connect via Bluetooth using Radiacode Serial (e.g. "RC-10x-xxxxxx").',
+    )
+    parser.add_argument(
+        '--bluetooth-uuid',
+        type=str,
+        required=False,
+        help='Connect via Bluetooth using Radiacode UUID (e.g. "11111111-2222-3333-4444-56789ABCDEF").',
+    )
+    parser.add_argument(
+        '--serial', type=str, required=False, help='Connect via USB using Radiacode Serial (e.g. "RC-10x-xxxxxx").'
+    )
+    parser.add_argument('--usb', type=str, required=False, help='(default) Connect via USB to the first Radiacode available.')
+    parser.add_argument('--interval', type=int, required=False, default=600, help='Send interval, seconds')
     args = parser.parse_args()
 
-    if args.connection == 'usb':
-        print('will use USB connection')
-        rc_conn = RadiaCode()
-    else:
-        print('will use Bluetooth connection')
-        rc_conn = RadiaCode(bluetooth_mac=args.bluetooth_mac)
+    try:
+        if args.bluetooth_mac:
+            print(f'Connecting to Radiacode via Bluetooth (MAC address: {args.bluetooth_mac})')
+            rc = RadiaCode(bluetooth_mac=args.bluetooth_mac)
+        elif args.bluetooth_uuid:
+            print(f'Connecting to Radiacode via Bluetooth (UUID: {args.bluetooth_uuid})')
+            rc = RadiaCode(bluetooth_uuid=args.bluetooth_uuid)
+        elif args.bluetooth_serial:
+            print(f'Connecting to Radiacode via Bluetooth (Serial: {args.bluetooth_serial})')
+            rc = RadiaCode(bluetooth_serial=args.bluetooth_serial)
+        elif args.serial:
+            print(f'Connecting to Radiacode via USB (Serial: {args.serial})')
+            rc = RadiaCode(serial_number=args.serial)
+        else:
+            print('Connecting to Radiacode via USB')
+            rc = RadiaCode()
+    except DeviceNotFoundBT as e:
```

Comment:
I think it's a bit overcomplicated with examples using mutiple `except smth as e: print(e)` - probably we want to just use generic `case Exception` everywhere or show all types of exceptions in only one example (basic.py?).